src/Microsoft.Azure.WebJobs/ParameterBindingData.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// A reference type for supporting SDK-type bindings in out-of-process Function workers
+    /// </summary>
+    public class ParameterBindingData
```

Comment:
Thinking we should mark this type "sealed" because we're relying on strict type equality and don't really support subclasses, right? Or do we want to allow for extensions to provide their own subclasses, e.g. ServiceBusParameterBindingData?

Reply:
I don't think there is any intent for subclasses, I can't think of a scenario right now where extension would need to build their own since the content is pretty flexible